Kuroba/app/src/main/java/com/github/adamantcheese/chan/ui/view/MultiImageView.java

```diff
@@ -103,45 +105,66 @@
 
     private Context context;
     private ImageView playView;
-    private GestureDetector exoDoubleTapDetector = new GestureDetector(context, new SimpleOnGestureListener() {
-        @Override
-        public boolean onDoubleTap(MotionEvent e) {
-            callback.onDoubleTap();
-            return true;
-        }
-    });
-    private GestureDetector gifDoubleTapDetector = new GestureDetector(context, new SimpleOnGestureListener() {
-        @Override
-        public boolean onDoubleTap(MotionEvent e) {
-            GifDrawable drawable = (GifDrawable) findGifImageView().getDrawable();
-            if (drawable.isPlaying()) {
-                drawable.pause();
-            } else {
-                drawable.start();
-            }
-            return true;
-        }
-
-        @Override
-        public boolean onSingleTapConfirmed(MotionEvent e) {
-            callback.onTap();
-            return true;
-        }
-    });
-
     private PostImage postImage;
     private Callback callback;
     private Mode mode = Mode.UNLOADED;
-
     private boolean hasContent = false;
     private ImageContainer thumbnailRequest;
     private FileCacheDownloader bigImageRequest;
     private FileCacheDownloader gifRequest;
     private FileCacheDownloader videoRequest;
-
     private SimpleExoPlayer exoPlayer;
-
     private boolean backgroundToggle;
+    private boolean imageAlreadySaved = false;
+
+    private static final float FLING_DIFF_Y_THRESHOLD = dp(60f);
+    private static final float FLING_VELOCITY_Y_THRESHOLD = dp(200f);
+    private static final float FLING_DIST_X_THRESHOLD = dp(100f);
```

Comment:
I think the horizontal devitaion should be like 60 or 70dp. 100 is kinda too much. it allows the gesture to deviate more to left/right. And it shouldn't.

Reply:
Done.